src/Analysis/Ast/Impl/Analyzer/Symbols/FunctionEvaluator.cs

```diff
@@ -19,6 +19,7 @@
 using System.Linq;
 using Microsoft.Python.Analysis.Analyzer.Evaluation;
 using Microsoft.Python.Analysis.Diagnostics;
+using Microsoft.Python.Analysis.Extensions;
```

Comment:
Namespace for extensions is `Microsoft.Python.Analysis`. We don't add separate namespace for them.

Reply:
Ok